dlt/sources/helpers/rest_client/client.py

```diff
@@ -99,8 +99,8 @@ def _create_request(
         self,
         path_or_url: str,
         method: HTTPMethod,
-        params: Optional[Dict[str, Any]] = None,
         headers: Optional[Dict[str, Any]] = None,
+        params: Optional[Dict[str, Any]] = None,
```

Comment:
we can revert this now, then the diff will get a bit smaller?

Reply:
reverted in dc89002c

Reply:
haha, that's why I did it in the original PR. I assumed it was just remnants. I can revert dc89002ce3569b0355be2f6c64923f6dce6cbbd0 again?

Reply:
it's ok, I have a fix for #2398 coming, and I'll fix it there

Reply:
sorry for the misinterpretation of intent.